includes/Ajax/Frontend_Form_Ajax.php

```
@@ -829,4 +870,343 @@
     }
+
+    /**
+     * Handle The Events Calendar event data using TEC's actual API
+     *
+     * @since WPUF_SINCE
+     *
+     * @param int  $post_id   The post ID
+     * @param bool $is_update Whether this is an update operation
+     * @return void
+     */
+    public function handle_events_calendar_data( $post_id, $is_update ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$is_update'. (Unused Code Rules)

(UnusedFormalParameter)